pages/en/billing.mdx

```diff
@@ -0,0 +1,76 @@
+
+- Add and remove GRT from your wallet
+- Keep track of your balances based on how much GRT you have added to your account, how much you have removed, and your invoices
+- Automatically clear payments based on query fees generated
+
+There are 4 states your invoice can be in:
+
+1. Created - your invoice has just been created and not been paid yet
+2. Paid - your invoice has been successfully paid
+3. Unpaid - there is not enough GRT in your balance on the billing contract
+4. Error - there is an error processing the payment
+
+**See the diagram below for more information:**
+
+![Billing Flow](/img/billing-flow.png)
+
+### Adding GRT to Account
+
+Before making queries on The Graph Network, you will need to add GRT to your account. To add GRT to your account, take the following steps:
+
+1. Purchase GRT and ETH on an exchange of your choice
+2. Send the GRT and ETH to your wallet
+3. Bridge GRT to Polygon using the Subgraph Studio UI
+
+   a) You will receive 0.001 Matic within a few minutes of sending GRT to the Polygon bridge. You can monitor the transaction by inputting your wallet address into the search bar on [Polygonscan](https://polygonscan.com/).
+
+4. Add bridged GRT to the billing contract on Polygon. The billing contract address is: [0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE](https://polygonscan.com/address/0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE).
+
+   a) In order to complete step #4, you'll need to switch your network in your wallet to Polygon. You can add Polygon's network by connecting your wallet and clicking on "Choose Polygon (Matic) Mainnet" [here](https://chainlist.org/). Once you've added the network, switch it over in your wallet by navigating to the network pill on the top right-hand side corner. In Metamask, the network is called **Polygon Mainnnet**.
+
+If you used your API keys over the course of any given week, you will receive an invoice based on the query fees you have generated during this period. This invoice will be paid using GRT available in your balance. Query volume is evaluated by the API keys you own. Your balance will be updated after fees are withdrawn.
+
+### Understanding Invoices in Subgraph Studio
+
+When using The Graph Network, dapps and users pay for query fees in GRT, the protocol's native work-utility token. Maintaining a positive balance for an account is the user's or dapp's responsibility.
+
+Here are ways to be notified about and/or check a query fee balance:
+
+- You can set up a billing alert that will be sent to an email address [here](https://thegraph.com/studio/settings/).
+
+- You can query the billing subgraph API directly [here](https://thegraph.com/hosted-service/subgraph/graphprotocol/billing?query=Nena%20account)
```

Comment:
let's remove "?query=Nena%20account" from this query please

Reply:
Got it - sorry about that!